paris/src/main/java/com/airbnb/paris/typed_array_wrappers/TypedArrayTypedArrayWrapper.kt

```diff
@@ -47,6 +51,19 @@ internal class TypedArrayTypedArrayWrapper constructor(
     override fun getFraction(index: Int, base: Int, pbase: Int): Float =
         typedArray.getFraction(index, base, pbase, -1f)
 
+    override fun getFont(index: Int): Typeface? {
+        return if (isNull(index)) {
+            null
+        } else {
+            val resourceId = getResourceId(index)
+            if (resourceId != 0) {
```

Comment:
Were you able to test that it returns `0` if the attribute value is a string? I would have expected an exception to be thrown.

Reply:
Yes. And also this is confirmed by javadocs:
```
* @param defValue Value to return if the attribute is not defined or
*                 not a resource.
```